packages/advisor-components/src/ReportDetails/ReportDetails.tsx

```diff
@@ -35,6 +35,11 @@ const ReportDetails: React.FC<ReportDetailsProps> = ({ report, kbaDetail, kbaLoa
     }
   };
 
+  const fixBrokenHydraLink = (url: string) => {
+    const linkToArray = url.split('/');
+    return `https://access.redhat.com/solutions/${linkToArray.at(-1)}`;
```

Comment:
I don't really know what is this supposed to be used for but, what about stage env. The stage uses `access.stage...`. Is this supposed to always hit prod?

Reply:
Links lead to the prod even on the stage env 
![image](https://user-images.githubusercontent.com/62722417/234275964-574f5c1b-747e-48f6-888b-6dea12cf8938.png)